### PR TITLE
Added typings.

### PR DESCRIPTION
**Pull request by VoidNulll**
Added some typings to AxonCore. Should be nice for anyone using typescript or just wanting to be able to have intellisense or something like that, that goes off of typings and not JSDoc

### CODE REVIEW

File: index.d.ts [hunk 1]
```diff
@@ -1,2 +1,552 @@
...
+        public infos?: ModuleInfo;
+
+        constructor(client: AxonClient);
+
+        // Init - External available
+        public init(commands: Command[], events: Event[], schema: object): void;
+        public initAllCommands(commands: Command[]): void;
+        public initSubCommands(subCommands: Command): void;
+        public initAllEvents(events: Event[]): void;
+        public initAllScehams(schemas: object): void;
```
Comment (Khaazz) [inline index.d.ts]: Typo here (Schemas)

File: index.d.ts [hunk 2]
```diff
@@ -1,2 +1,552 @@
...
+        public unregisterSchema(label: string): boolean;
+        public unregisterEvent(label: string): boolean;
+    }
+
+    export class DBService {
+        constructor();
+        public fetchAxon(): Promise<object> | null;
+        public fetchGuild(gID: string): Promise<object> | null;
+        public initAxon(): Promise<object>;
+        public initaGuild(axonClient: AxonClient, gID: string): Promise<object> | null;
```
Comment (Khaazz) [inline index.d.ts]: typo here as well (initGuild)

File: package.json [hunk 3]
```diff
@@ -9,6 +9,7 @@
     "node": ">=8.0.0"
   },
   "dependencies": {
+    "@types/node": "^12.0.10",
```
Comment (Khaazz) [inline package.json]: can't that be a dev dependency? (I honestly don't know)

**Comment by VoidNulll** [comment]
Updated..

For some reason i rebased on the upstream repo/dev, idk.

**Pull request closed without merging**